### PR TITLE
Feature/firstversion

### PR DESCRIPTION
**Pull request by marians**
This is the version I'd like to publish alongside the tutorial.

@luebken Happy for pair of eyeballs.

(TODO: change README.md to contain the link to the published tutorial)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright 2015 Giant Swarm GmbH
```
Comment (zeisss) [inline LICENSE:1]: Why Apache 2.0? We used MIT for everything so far. Any reasons?

File: backup.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# Amazon S3 target bucket
+bucket=$S3_BUCKET
+
+# pattern to create subdirectories from dateelements,
+# e. g. '%Y/%m/%d' or '%Y/%Y-%m-%d'
+pathpattern=$PATH_DATEPATTERN
+
+# initially wait a minute before first backup
+#sleep 60
+sleep 1
```
Comment (zeisss) [inline backup.sh]: Comment and code differ. Propably just a change for testing

File: swarm.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      ]
+    },
+    {
+      "service_name": "archiver",
+      "components": [
+        {
+          "component_name": "mysql-archiver",
+          "image": "registry.giantswarm.io/yourusername/mysql-archiver:latest",
+          "env": {
+            "DB_NAME": "$mysqldb",
```
Comment (zeisss) [inline swarm.json:31]: Do we want to add a `swarmvars.json.example` file to show the format?

File: LICENSE (hunk 1 above)
Comment (marians) [inline LICENSE:1]: See internal slack discussion

File: backup.sh (hunk 2 above)
Comment (marians) [inline backup.sh]: Fixed

File: swarm.json (hunk 3 above)
Comment (marians) [inline swarm.json:31]: That would require to change the guide content. Just having it in the repository without explanation wouldn't do, I guess. Maybe I add this later.

**Pull request merged**